nim.cfg

```diff
@@ -0,0 +1,2 @@
+# Nim compilation defaults for the whole repository
+--gc: arc
```

Comment:
Why did you put these in a new nim.cfg file?
I think that .nims file superseded nim.cfg files: see e.g. https://forum.nim-lang.org/t/6844#42785

Reply:
This is a workaround for a bug in stable nim:
https://github.com/TheAlgorithms/Nim/pull/21#issuecomment-1574393251

My bad, turns out I was on devel when I checked this.

Reply:
I do not understand. Had you only the error on your devel version? Please, open an issue to remember to fall back entirely to .nims configuration after Nim 2.0 release.

Reply:
On the contrary. I was on devel and missed this error. I'll open the issue when I (or someone else) identify the fixing PR for Nim to watch it landing in stable.